### PR TITLE
BTA-1451 : navigation regions confusing Home - Manage account - Message - Help and contact Nav is not main content

### PR DESCRIPTION
**Pull request by nehakannaujia1**
BTA-1451 : navigation regions confusing Home - Manage account - Message - Help and contact Nav is not main content

### CODE REVIEW

File: app/views/govuk_wrapper.scala.html [hunk 1]
```diff
@@ -87,29 +87,44 @@
...
-        @views.html.language_selection(
-        appConfig.languageMap,
-        appConfig.routeToSwitchLanguage,
-        Some("u-align--right"))
-    }
-    @if(contentHeader.isDefined) {
-        @uiLayouts.main_content_header(contentHeader = contentHeader.get)
-    }
+@getHelpForm = @{uiHelpers.reportAProblemLink(appConfig.reportAProblemPartialUrl, appConfig.reportAProblemNonJSUrl)}
+
```
Comment (gianick) [inline app/views/govuk_wrapper.scala.html]:
```suggestion
```

File: app/views/govuk_wrapper.scala.html [hunk 2]
```diff
@@ -87,29 +87,44 @@
...
+
```
Comment (gianick) [inline app/views/govuk_wrapper.scala.html]:
```suggestion
```

File: app/views/govuk_wrapper.scala.html [hunk 3]
```diff
@@ -87,29 +87,44 @@
...
+    @sidebar
+
+    @getHelpForm
+
+  </div>
+</main>
 }
 
+
```
Comment (gianick) [inline app/views/govuk_wrapper.scala.html]:
```suggestion
```

**Comment by nehakannaujia1** [comment]
@gianick  : Changes done :-)

**Pull request merged**